### PR TITLE
fix typos in some code and comment

### PR DESCRIPTION
**Pull request by yzhao244**
Minor issue, I just found a few typos on db_test and column_family while reading the code. And I have this PR opened to contribute.  :)

### CODE REVIEW

**Comment by japui** [comment]
corrija erros de digitação em algum código e comente #10139

**Comment by yzhao244** [comment]
@siying @ajkr Hi guys, this is my first PR for rocksdb community. :) .. not too sure about the PR review process. Do I have to manually add one of you guys to be the reviewer of the PR?Or I just wait lol..

File: db/db_test.cc [hunk 1]
```diff
@@ -646,7 +646,7 @@ TEST_F(DBTest, SingleDeleteFlush) {
 
     ASSERT_EQ("NOT_FOUND", Get(1, "bar"));
     ASSERT_EQ("NOT_FOUND", Get(1, "foo"));
-    // Skip FIFO and universal compaction beccause they do not apply to the test
+    // Skip FIFO and universal compaction beccaus they do not apply to the test
```
Comment (siying) [inline db/db_test.cc:649]: Still appears to be incorrect? :)

**Review by siying — COMMENTED** [review]
Thanks for the contribution.

File: db/db_test.cc (hunk 1 above)
Comment (yzhao244) [reply db/db_test.cc:649]: lol, good catch, fixed my typo :)

Comment (siying) [reply db/db_test.cc:649]: Did you forget to update the PR?

Comment (yzhao244) [reply db/db_test.cc:649]: @siying oh, my second commit fixed the typo.

**Comment by yzhao244** [comment]
@siying addressed the review comment, should be good this time.

**Comment by yzhao244** [comment]
@siying any updates on merging :)

**Pull request closed without merging**